p2p/transport/tcp/conn/connection.go

```diff
@@ -98,10 +102,13 @@ type MConnection struct {
 
 	_maxPacketMsgSize int
 
-	// streamID -> list of incoming messages
-	recvMsgsByStreamID map[byte]chan []byte
```

Comment:
I guess this addresses the concern from this [comment](https://github.com/cometbft/cometbft/pull/4307/files#r1850397779) ?

Reply:
yes